grafana-plugin/src/containers/EscalationChainSteps/EscalationChainSteps.tsx

```diff
@@ -26,10 +28,19 @@ interface EscalationChainStepsProps {
   offset?: number;
 }
 
+const getStyles = (_theme: GrafanaTheme2) => {
```

Comment:
underscore suggest as if it's unused param, but it is used, can we change it to just `theme`?

Reply:
Good catch